src/components/pill/pill.scss

```diff
@@ -6,6 +6,7 @@
   font-size: 12px;
   font-weight: 700;
   letter-spacing: 0.7px;
+  line-height: 12px;
```

Comment:
Although `12px` was the original value before the refactor, it looks like this is too low to resolve the requirement on the ticket of "Expected: Status indicator 'pills' are generally 21px in height.", it looks like `15px` works better.

I wonder if we should target this change against the release branch as well so we can ship it sooner.